sdk/eventhub/event-hubs/samples/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "azure-event-hubs-samples",
```

Comment:
Thoughts on adding `"private": true,` so that this doesn't accidentally end up getting published someday?

Reply:
+1 on this idea.